events-api/src/database/migrations/1727152878073-create-tickets.sql

```diff
@@ -0,0 +1,11 @@
+-- UP
+CREATE TABLE IF NOT EXISTS tickets (
+    ticket_id TEXT PRIMARY KEY,  -- Corrected from 'ticked_id'
+    ticket_quantity INTEGER NOT NULL,
+    ticket_price DECIMAL NOT NULL,
+    event_id TEXT NOT NULL,
+    FOREIGN KEY (event_id) REFERENCES events(event_id)
```

Comment:
I think that we need here to specify what happen if event was deleted.

Reply:
Please resolve this

Reply:
![image](https://github.com/user-attachments/assets/1ede8b84-3682-4241-988d-6550648cf9a9)

Reply:
but still not deleted the event if it is used as forign key 
therefore I deleted manually related tickets then event itself
![image](https://github.com/user-attachments/assets/fc178a81-8f31-41c1-b9ce-9cfd02863087)